.github/workflows/mediasoup-node.yaml

```diff
@@ -11,12 +11,22 @@ concurrency:
 jobs:
   ci:
     strategy:
+      fail-fast: false
```

Comment:
While this is good while writing a PR we prefer not to have this once merged so I will remove this once we merge this PR.